include/swift/SILOptimizer/Analysis/RegionAnalysis.h

```diff
@@ -357,15 +357,15 @@ class RegionAnalysisValueMap {
 public:
   RegionAnalysisValueMap(SILFunction *fn) : fn(fn) { }
 
-  /// Returns the value for this instruction if it isn't a fake "represenative
+  /// Returns the value for this instruction if it isn't a "fake" representative
```

Comment:
The change to quotation marks here is incorrect.

Reply:
It is consistent with the rest in the file. Many `"fake"` occurrences. But i will revert/take another look!

Reply:
fixed